prime-router/src/main/resources/db/migration/V55__add_normalized_code_values_elr_metadata.sql

```diff
@@ -0,0 +1,151 @@
+        WHEN '100892-9' THEN 'Orthopoxvirus IgM'
+    END
+;
+
+UPDATE
+    elr_result_metadata
+SET
+    test_ordered_long_name = case test_ordered_code
+        WHEN '41853-3' THEN 'Orthopoxvirus DNA [Presence] in Specimen by NAA with probe detection'
+        WHEN '100434-0' THEN 'Orthopoxvirus non-variola DNA [Presence] in Specimen by NAA with probe detection'
+        WHEN '100383-9' THEN 'Monkeypox virus DNA [Presence] in Specimen by NAA with probe detection'
+        WHEN '100888-7' THEN 'West African monkeypox virus DNA [Presence] in Specimen by NAA with probe detection'
+        WHEN '100889-5' THEN 'Congo Basin monkeypox virus DNA [Presence] in Specimen by NAA with probe detection'
+        WHEN '100885-3' THEN 'Parapoxvirus DNA [Presence] in Specimen by NAA with probe detection'
+        WHEN '100886-1' THEN 'Orf virus DNA [Presence] in Specimen by NAA with probe detection'
+        WHEN '100887-9' THEN 'Pseudocowpox virus DNA [Presence] in Specimen by NAA with probe detection'
+        WHEN '100891-1' THEN 'Orthopoxvirus IgG Ab [Presence] in Serum or Plasma by Immunoassay'
+        WHEN '100892-9' THEN 'Orthopoxvirus IgM Ab [Presence] in Serum or Plasma by Immunoassay'
+END
+;
+
+UPDATE
+    elr_result_metadata
+SET
+    test_performed_normalized = case test_performed_code
+        WHEN '41853-3' THEN 'Orthopoxvirus'
+        WHEN '100434-0' THEN 'Non-variola Orthopoxvirus'
+        WHEN '100383-9' THEN 'Monkeypox Virus'
+        WHEN '100888-7' THEN 'West African Monkeypox Virus'
+        WHEN '100889-5' THEN 'Congo Basin Monkeypox Virus'
+        WHEN '100885-3' THEN 'Parapoxvirus'
+        WHEN '100886-1' THEN 'Orf Virus'
+        WHEN '100887-9' THEN 'Pseudocowpox Virus'
+        WHEN '100891-1' THEN 'Orthopoxvirus IgG'
+        WHEN '100892-9' THEN 'Orthopoxvirus IgM'
+    END
+;
+
+UPDATE
+    elr_result_metadata
+SET
+    test_performed_long_name = case test_performed_code
+        WHEN '41853-3' THEN 'Orthopoxvirus DNA [Presence] in Specimen by NAA with probe detection'
+        WHEN '100434-0' THEN 'Orthopoxvirus non-variola DNA [Presence] in Specimen by NAA with probe detection'
+        WHEN '100383-9' THEN 'Monkeypox virus DNA [Presence] in Specimen by NAA with probe detection'
+        WHEN '100888-7' THEN 'West African monkeypox virus DNA [Presence] in Specimen by NAA with probe detection'
+        WHEN '100889-5' THEN 'Congo Basin monkeypox virus DNA [Presence] in Specimen by NAA with probe detection'
+        WHEN '100885-3' THEN 'Parapoxvirus DNA [Presence] in Specimen by NAA with probe detection'
+        WHEN '100886-1' THEN 'Orf virus DNA [Presence] in Specimen by NAA with probe detection'
+        WHEN '100887-9' THEN 'Pseudocowpox virus DNA [Presence] in Specimen by NAA with probe detection'
+        WHEN '100891-1' THEN 'Orthopoxvirus IgG Ab [Presence] in Serum or Plasma by Immunoassay'
+        WHEN '100892-9' THEN 'Orthopoxvirus IgM Ab [Presence] in Serum or Plasma by Immunoassay'
+    END
+;
```

Comment:
All of the updates are to fill in historical data that did not have these columns available. There's only ~500 recs in the table now, so these should be fast, and only need to happen once.